{"run_number": "1683763096057601", "step_name": "process", "task_id": "3", "attempt_id": 0, "name": "_exception", "content_type": "gzip+pickle-v2", "type": "metaflow.artifact", "sha": "f3627f46179fdd95bf0e83101840fd1d71b60e40", "ds_type": "local", "location": ":root:/Users/ajay/Downloads/mlops/credit_fraud/monitoring/.metaflow", "flow_id": "CreditClassifierMonitor", "user_name": "ajay", "tags": [], "system_tags": ["python_version:3.10.9", "runtime:dev", "user:ajay", "metaflow_version:2.8.5"], "ts_epoch": 1683763098426}